src/core/reducers/installations.js

```diff
@@ -67,12 +104,12 @@ export default function installations(
       return {
         ...state,
         [payload.guid]: {
-          guid: payload.guid,
-          url: payload.url,
-          error: payload.error,
           downloadProgress: 0,
-          status: payload.status,
+          error: payload.error,
+          guid: payload.guid,
           needsRestart: payload.needsRestart || false,
+          status: payload.status,
+          url: payload.url,
```

Comment:
Sorry for that, I reordered the list of props for another patch (to better see what I was doing)